app/src/tasks/view/tasksDisplay.html

```diff
@@ -0,0 +1,34 @@
+<md-content layout="column">
+  
+    <div class="sub-toolbar" layout="row" layout-align="space-between center" layout-padding>
+    
+        <h1 class="md-headline" layout-margin>{{ 'TASK.TASKS' | translate }}</h1>
+
+        <md-select ng-model="status" >
+            <md-option value="" selected>Show all open tasks</md-option>
+            <md-option ng-repeat="status in tasksCtrl.statuses" value="{{status}}">{{status | workflowTaskStatus}}</md-option>
+        </md-select>
+    
+    </div>
+
+    <div layout="column">
+        <md-card ng-repeat="task in tasksCtrl.tasks | filter:status">
+            <md-card-content>
+              <div style="text-align: right; font-size: 14px;" class="muted">
+                  {{'COMMON.CREATED_ON' | translate}} {{task.properties.cm_created | date}} {{'COMMON.BY' | translate}} {{task.workflowInstance.initiator.firstName + ' ' + task.workflowInstance.initiator.lastName}}
```

Comment:
I suggest to merge into 1 translation item, because structure in other languages be different.
eg. in caseInfolet.html: 
```
{{ 'CASE.MODIFIED_BY' | translate: {at:$filter('openeDate')(case["cm:modified"].value), by:case["cm:modifier"].fullname} }}
```
* $filter - must be passed to controller (maybe you could find a nicer way :) )
json:
```
"MODIFIED_BY": "Last modified {{at}} by {{by}}"
```